ibazel/ibazel.go

```diff
@@ -26,7 +26,9 @@ import (
 
 	"github.com/bazelbuild/bazel-watcher/bazel"
 	"github.com/bazelbuild/bazel-watcher/ibazel/command"
+	"github.com/bazelbuild/bazel-watcher/ibazel/lifecycle"
```

Comment:
This was needed to avoid circular deps. Since I'm passing the Lifecycle interface into some of the hooks, the lifecycle listeners needed to have access to the interface.